<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppletServiceBaseClass</class>
 <widget class="QWidget" name="AppletServiceBaseClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>301</width>
    <height>549</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Group Host Service</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="m_ServiceTitleLabel">
     <property name="text">
      <string>Service Title</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="InformationWidget" name="m_InfoWidget" native="true"/>
   </item>
   <item>
    <widget class="QLabel" name="m_PermissionLabel">
     <property name="text">
      <string>Permission Level Required To Use Service</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="PermissionWidget" name="m_PermissionWidget" native="true"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="m_StartButton">
       <property name="text">
        <string>Start Service</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_StopButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Stop Service</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="m_UrlDescriptionLabel">
     <property name="text">
      <string>Service URL or IP Address</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="m_UrlEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="m_UserDisplayedNameLabel">
     <property name="text">
      <string>Name Of Service ( Optional For Detail Information )</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="m_NameEdit">
     <property name="maxLength">
      <number>127</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="m_DescriptionLabel">
     <property name="text">
      <string>Description Of Service ( Optional For Detail Information )
( max 511 characters )</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="m_DescriptionEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="LogWidget" name="m_LogWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PermissionWidget</class>
   <extends>QWidget</extends>
   <header>PermissionWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>InformationWidget</class>
   <extends>QWidget</extends>
   <header>InformationWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LogWidget</class>
   <extends>QWidget</extends>
   <header>LogWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../gotvcommon.qrc"/>
 </resources>
 <connections/>
</ui>
